addons/sale/views/product_product_views.xml

```diff
@@ -63,7 +63,7 @@
                        groups="product.group_product_variant"/>
                 <field name="product_tmpl_id" string="Product Template"/>
                 <!-- Filter -->
-                <filter string="Favorites" name="favorites" domain="[('priority', '= ','1')]"/>
+                <filter string="Favorites" name="favorites" domain="[('priority', '=', '1')]"/>
```

Comment:
To fix in stable version.

Reply:
https://github.com/odoo/odoo/pull/138376